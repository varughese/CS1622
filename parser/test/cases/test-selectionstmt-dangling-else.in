void main(void)
{
    if(1 == 1) if(2 == 2) 3+3; else 4+4;
}
