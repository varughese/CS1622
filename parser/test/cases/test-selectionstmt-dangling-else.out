[program
  [fun-declaration
    [void]
    [main]
    [params]
    [compound-stmt 
      [selection-stmt 
        [==  [1] [1]]
        [selection-stmt 
          [==  [2] [2]]
          [+  [3] [3]]
          [+  [4] [4]]
        ]
      ]
    ]
  ]
]
